experiments/batch-rca-automation/batch_rca_headless.sh

```
@@ -0,0 +1,306 @@
+# Extract env vars from JSON using python
+eval $(python3 -c "
+import json, sys
+try:
+    with open('$SETTINGS_FILE') as f:
+        settings = json.load(f)
+    for key, value in settings.get('env', {}).items():
+        print(f'export {key}=\"{value}\"')
+except Exception as e:
+    print(f'echo \"[ERROR] Failed to load settings.json: {e}\"', file=sys.stderr)
+    sys.exit(1)
+")
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)